lib/configs/angular.js

```diff
@@ -99,10 +98,16 @@ module.exports = {
         }],
         'space-before-blocks': [2, "always"],
         'consistent-this': [2, "self"],
-        'indent': [2, 4, {
-            SwitchCase: 1
-        }],
-        quotes: [2, "single"]
+        quotes: [2, "single"],
+        'prefer-const': 1,
+        'prefer-template': 1,
```

Comment:
@WesTyler @dhinklexo So even with it set to 1, this rule will make it so that running `eslint --fix` will replace concatenation with template strings. I didn't this would be problematic in fe-tkp because we're already using webpack with babel to polyfill, but from what you're saying that might be incorrect?